src/assets/data/team.json

```
@@ -8,15 +8,15 @@
   {
     "img": "/src/assets/img/team/Jonas Wanke.webp",
     "name": "Jonas Wanke",
-    "lastRole": { "en": "Expert", "de": "Experte" },
-    "skill": "Skill 08",
+    "lastRole": { "en": "Expert", "de": "Bundestrainer" },
+    "skill": "Mobile Applications Development",
```

The skill names also need to be localized:

```suggestion
    "skill": {
      "de": "App-Entwicklung",
      "en": "Mobile Applications Development"
    },
```

---

How does this apply to the other skills? Are they fine the way they are or do we need localized names for them as well?